modules/claims_api/app/models/claims_api/power_of_attorney.rb

```diff
@@ -13,17 +13,18 @@ class PowerOfAttorney < ApplicationRecord
 
     PENDING = 'pending'
     UPDATED = 'updated'
+    SUBMITTED = 'submitted'
```

Comment:
this was the only state not captured with a constant, so I just added that here, as well